p2p/pex/pex_reactor.go

```diff
@@ -551,7 +562,9 @@ func (r *PEXReactor) dialPeer(addr *p2p.NetAddress) error {
 		default:
 			r.attemptsToDial.Store(addr.DialString(), _attemptsToDial{attempts + 1, time.Now()})
 		}
-		return errors.Wrapf(err, "dialing failed (attempts: %d)", attempts+1)
+		return errors.Wrapf(err, "dialing failed (attempts: %d,"+
+			"persistent_peers_max_dial_period: %s, persistent_peer: %t)", attempts+1,
```

Comment:
don't think we should add config params here because they are not specific to this error

Reply:
I see, I'll revert this